README.md

```diff
@@ -291,12 +351,15 @@ require 'my_module'
 # Create a client
 client = Temporalio::Client.connect('localhost:7233', 'my-namespace')
 
-# Create a worker with the client and activities
+# Create a worker with the client, activities, and workflows
 worker = Temporalio::Worker.new(
   client:,
   task_queue: 'my-task-queue',
-  # There are various forms an activity can take, see specific section for details.
-  activities: [MyModule::MyActivity]
+  workflows: [MyModule::MyWorkflow],
+  # There are various forms an activity can take, see "Activities" section for details
+  activities: [MyModule::MyActivity],
+  # During the beta period, this must be provided explicitly, see below for details
```

Comment:
Not sure about the "beta period" expression (it is used in a few place in this README file). It is not part of our usual terminology, and it's not clear to me how that would map to our actual version scheme. And anyway, is it really bound to a specific release, like that will be this way until 1.0, then change from that point on? Or is it just something not yet implemented? If the later, might just say "For now".

Reply:
The concept of a "beta" library is well known to developers, especially Ruby ones. We need to decide what to do with this default by 1.0. This will not be a required field in 1.0, but we are not sure what will be the default yet, so we are requiring it be explicitly set.